cloudwatch-logs-subscription-firehose-cdk/bin/cloudwatch-logs-subscription-firehose-cdk.ts

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env node
+import 'source-map-support/register';
+import * as cdk from 'aws-cdk-lib';
+import { CloudwatchLogsSubscriptionFirehoseCdkStack } from '../lib/cloudwatch-logs-subscription-firehose-cdk-stack';
```

Comment:
Think we are missing the lib folder here?

Reply:
Sorry about this @boyney123 
Forgot to define it in gitignore. This is fixed now.

Reply:
Thanks!